galsim/transform.py

```diff
@@ -20,7 +20,7 @@
 """
 
 import galsim
-import numpy
+import numpy as np
```

Comment:
I think this one is standard enough to leave.  Mostly though, I'd just like this to be consistent so when I write new code I don't have to lookup if it's going to be `numpy` or `np`.